packages/config/config/devices/0x0345/templates/swidget_template.json

```diff
@@ -0,0 +1,1504 @@
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 300
+	},
+	"motion_reset_delay": {
+		// param 50
+		"label": "Motion Reset Delay",
+		"description": "Used to set the time in seconds between optionally resending NOTIFICATION REPORT",
+		"valueSize": 2,
+		"minValue": 1,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 300
+	},
+	"motion_detected_entry_set_value": {
+		// param 51
+		"label": "Motion Detected Entry SET Value",
+		"description": "BASIC SET value sent to associated nodes on entry into motion detected state (Occupied)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 255
+	},
+	"motion_detected_exit_set_value": {
+		// param 52
+		"label": "Motion Detected Exit SET Value",
+		"description": "BASIC SET value sent to associated nodes on exit from motion detected state (Vacant)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"motion_actions_control_associations_on_entry": {
+		// param 53b0
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL ASSOCIATIONS ON ENTRY",
+		"description": "If set, send BASIC SET with “Motion Detected Entry SET Value” to “Motion” association group on entry into motion detected state (Occupied)"
+	},
+	"motion_actions_control_associations_on_exit": {
+		// param 54b1
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL ASSOCIATIONS ON EXIT",
+		"description": "If set, send BASIC SET with “Motion Detected Exit SET Value” to “Motion” association group on exit from motion detected state (Vacant)"
+	},
+	"motion_actions_control_host_on_entry": {
+		// param 54b2
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL HOST ON ENTRY",
+		"description": "If set, execute BASIC SET with “Motion Detected Entry SET Value” on host device on entry into motion detected state (Occupied)"
+	},
+	"motion_actions_control_host_on_exit": {
+		// param 54b3
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: CONTROL HOST ON EXIT",
+		"description": "If set, execute BASIC SET with “Motion Detected Exit SET Value” on host device on exit from motion detected state (Vacant)."
+	},
+	"motion_actions_report_transitions": {
+		// param 54b4
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Motion Actions: NOTIFY TRANSITIONS",
+		"description": "If set, execute BASIC SET with “Motion Detected Exit SET Value” on host device on exit from motion detected state (Vacant)."
+	},
+    "air_quality_level_1_index": {
+		// param 64
+		"label": "Air Quality Level 1 Index",
+		"description": "Used to set the index value representing the upper limit for the 'Clean' air state.",
```

Comment:
```suggestion
		"label": "Air Quality Threshold: Clean",
```

Reply:
These are the parameter names per the manufacturers zwave documentation. I would prefer to keep them in-sync.